resources/views/dashboard/index.blade.php

```diff
@@ -2,29 +2,64 @@
 
 @section('body_class', 'body__home')
 
-@section('content')
-
-    <call-to-action></call-to-action>
+@section('styles')
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@splidejs/splide@latest/dist/css/splide.min.css">
```

Comment:
Doesn't look like we are using this, right?

Reply:
Yep removing thanks